sdk/src/test/java/io/customer/sdk/CustomerIOTest.kt

```diff
@@ -156,6 +158,55 @@ internal class CustomerIOTest : BaseTest() {
         verifySuccess(response, Unit)
     }
 
+    @Test
+    fun `verify SDK both default and custom attributes are added`() {
+
+        `when`(
+            mockCustomerIO.api.registerDeviceToken(
+                any(),
+                any(),
+            )
+        ).thenReturn(getEmptyAction())
+
+        customerIO.deviceAttributes.putAll(mapOf("test" to "value"))
+
+        val expectedToken = "token"
+
+        val expectedAttributes = customerIO.deviceAttributes + deviceStore.buildDeviceAttributes()
+
+        val response = customerIO.registerDeviceToken(expectedToken).execute()
+
+        verify(mockCustomerIO.api).registerDeviceToken(expectedToken, expectedAttributes)
+
+        verifySuccess(response, Unit)
+    }
+
+    @Test
+    fun `verify SDK skips default attributes on basis of config`() {
```

Comment:
In the background queue PR, I moved tests from `test/` into `sharedTest/` so that they can be run from JVM or Android OS. Since our SDK at that point does more operations on the OS, it's a good idea to test on Android OS as well as JVM. With this move, test function naming does not allow the syntax of using the ` character in it. If you could, please, rename test function names to snakeCase that would save us time to refactor in the future. 

No need to rename *all* test function names. Just for the test functions you have created. 

Same with `should be equal to` needs to be snakeCase, too.